dev/config/pyramid_oereb.yml.mako

```diff
@@ -44,7 +44,7 @@ pyramid_oereb:
     # service and returns the result to the user.
 % if print_backend == 'XML2PDF':
     # Configuration for XML2PDF print service
-    renderer: pyramid_oereb.contrib.print_proxy.xml_2_pdf.Renderer
+    renderer: pyramid_oereb.contrib.print_proxy.xml_2_pdf.xml_2_pdf.Renderer
```

Comment:
@jwkaltz is that right? `.xml_2_pdf.xml_2_pdf.Renderer`

Reply:
yes, since the modules were refactored, it is the same as .mapfish_print.mapfish_print.Renderer in https://github.com/openoereb/pyramid_oereb/blob/check_xml2pdf/dev/config/pyramid_oereb.yml.mako#L62in